### PR TITLE
Add camel-spring-security wrapper

### PR DESCRIPTION
**Pull request by lopushen**

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -2045,6 +2045,12 @@
         <bundle dependency='true'>wrap:mvn:org.springframework.data/spring-data-redis/${spring-data-redis-version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-spring-redis/${project.version}</bundle>
     </feature>
+    <feature name='camel-spring-security' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='${camel.osgi.spring.version}'>spring</feature>
+        <bundle dependency='true'>wrap:mvn:org.springframework.security/spring-security-core/${spring-version}</bundle>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml]: the dependency tree suggests version `6.2.3` and the `spring-version` we use is 6.1.5

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: Yes , you can use ${spring-security-version}

Comment (lopushen) [reply features/src/main/feature/camel-features.xml]: fixed, thanks

**Review by f2par0 — APPROVED** [review]
- no conflict :heavy_check_mark: 
- compilation works :heavy_check_mark: 
- no unnecessary usage of wrap protocol :heavy_check_mark: 
- no unnecessary dependencies :heavy_check_mark: 
- no hardcoded version if it's in the parent pom :heavy_check_mark: 
- feature is placed in alphabetical order in the file camel-features.xml  :heavy_check_mark: 
- usage of dependent features (netty, jackson, jakarta ...) :heavy_check_mark:

**Review by essobedo — APPROVED** [review]

**Pull request merged**